automotive/build.gradle.kts

```diff
@@ -0,0 +1,234 @@
+import com.github.triplet.gradle.androidpublisher.ResolutionStrategy
+import com.google.gms.googleservices.GoogleServicesPlugin.GoogleServicesPluginConfig
+
+plugins {
+    id("com.android.application")
+    id("kotlin-android")
+    id("kotlin-kapt")
+    id("kotlin-parcelize")
+    id("com.google.firebase.appdistribution")
+    id("com.github.triplet.play")
+    id("com.google.gms.google-services")
+    kotlin("kapt")
+    id("dagger.hilt.android.plugin")
+}
+
+android {
+    namespace = "io.homeassistant.companion.android"
+
+    compileSdk = 33
+
+    ndkVersion = "21.3.6528147"
+
+    useLibrary("android.car")
+
+    defaultConfig {
+        applicationId = "io.homeassistant.companion.android"
+        minSdk = 29
+        targetSdk = 33
+
+        versionName = System.getenv("VERSION") ?: "LOCAL"
+        // We add 2 because the app, wear (+1) and automotive versions need to have different version codes.
+        versionCode = (System.getenv("VERSION_CODE")?.toIntOrNull() ?: 1) + 3
```

Comment:
So we need the `VERSION_CODE` variable increased by 3 in the pipeline too.  Otherwise we are going to conflict.

Reply:
Done